### ISSUE TITLE
Accessibility bug: Hidden Asides headings in sidebar.php

### ISSUE THREAD

**Issue by LukaszJaro** [wprig/wprig#761]
https://github.com/wprig/wprig/blob/master/sidebar.php has a hidden heading for screen readers:

`<h2 class="screen-reader-text"><?php esc_attr_e( 'Asides', 'wp-rig' ); ?></h2>`

Recommended to remove since it does not describe the sections content or purpose.

**Comment by dthenley** [comment]
While I do agree that the Asides title isn't going to be relevant at all for anybody project I do like that it is there and will keep the developer thinking to change it to what it will need to be used for. Maybe we can change it 'Asides' to 'Blog Sidebar' to give the user a better understanding when using a screen reader to navigate an unchanged wprig theme, while still leaving that placeholder for any developer to use in the future.

**Comment by dthenley** [comment]
@LukaszJaro I think a good middle ground would be to make it just 'Sidebar' that way it comes non-opiniated but still gives better context then asides. I made a PR for it

**Comment by LukaszJaro** [comment]
Thanks @dthenley for looking into this. I still think it should be removed completely as even adding the hidden screen reader heading is still being opinionated.

In my case, the asides normally include a visible heading so I would not require a separate hidden screen reader heading. Currently asides are announced as role complementary, if you want to specifically label them best practice appears to be using aria-label:

```
<aside id="sidebar" aria-label="Sponsors">
  <!-- content -->
</aside>
```

Example of how this would look:

![image](https://user-images.githubusercontent.com/10998875/148258212-41e60420-0d64-4658-b3df-b301f773e848.png)

Leaving out the the hidden heading will probably be most helpful, especially since many will not remove it and may end up with duplicate headings resulting in screen reader noise.

Here's reference https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/complementary_role
